content/getting-started/livechat-apps/webhook-apps/index.mdx

```diff
@@ -9,102 +9,121 @@ desc: "Learn how to build your own integration using LiveChat webhooks."
+To build an integration, you'll need an account in the LiveChat Developer Console. [Sign up here](https://developers.livechatinc.com/console/) and get access to all Platform Developer Tools.
+
+## 2. Create a new app
 
-## 2. Create new app
+Once you register an account, create a new application in the Developer Console using the `Create new app` button.
 
 ![LiveChat Developer Console](./images/livechat-developers-console-create-app.png)
 
-## 3. Add Settings Page
+## 3. Add Agent App Widgets block
 
-Go to Building Blocks and create a new Agent App Widget with `App Settings` placement. This widget will be displayed as a settings page for your app.
+Go to the Building Blocks section and create a new Agent App Widget with `App Settings` as the placement. This widget will be displayed as a settings page for your app.
 
 ![Create new Settings widget](./images/livechat-developers-console-create-widget.png)
 
-## 4. Add Authorization Block
+## 4. Add the App Authorization block
 
-Go back to Building Blocks section and create a new `Server-side app` Authorization Block. Save the highlighted `Client ID` &ndash; you'll need it later!
+Go back to the Building Blocks section and add the `Server-side app` option. Save the highlighted `Client Id` &ndash; you'll need it later to complete the authorization.
 
 ![Create new server-side Authorization block](./images/livechat-developers-console-create-auth.png)
 
 ## 5. Create a settings page
 
-Settings page is the place where you can:
+Settings page is a place where you can:
 
-- ask users to connect their accounts with your OAuth2.0 provider,
-- display configuration options,
-- inform users about additional installation steps (if there are any).
+- Prompt users to connect their accounts with your OAuth2.1 provider.
+- Display configuration options.
+- Inform users about additional installation steps (if there are any).
 
-If you need to access LiveChat user data, we recommend using [Sign in with LiveChat SDK](/getting-started/authorization/sign-in-with-livechat/).
+If you need to access LiveChat user data, we recommend using [Sign in with LiveChat authorization flow](/getting-started/authorization/sign-in-with-livechat/). For more information on possible widget placements, visit the [app locations](/extending-agent-app/#app-locations) page.
 
 ### Sample settings page scenario
 
-For instance if you're building an integration that binds LiveChat data with an external service, you should create a settings page that:
+If you're building an integration that binds LiveChat data with an external service, you should create a settings page that:
 
-1. Imports [Sign in with LiveChat SDK](/getting-started/authorization/sign-in-with-livechat/).
+1. Makes use of the [Sign in with LiveChat flow](/getting-started/authorization/sign-in-with-livechat/).
 2. Imports your sign in form or other authorization flow (e.g. "Connect with..." button).
```

Comment:
+ link to https://developers.livechat.com/docs/authorization/authorizing-api-calls/#authorization-code-grant